src/platform/site-wide/sass/formation-overrides/_functions.scss

```diff
@@ -0,0 +1,15 @@
+
+@function is-rem($value) {
+  @if unit($value) == "rem" {
+    @return true;
+  }
+}
+
+@function uswds-override($original) {
+  @if is-rem($original) {
+    $rem-to-px: $original * 10; // Formation base
+    $px-to-rem: $rem-to-px / 16; // USWDS base
+    @return $px-to-rem; // return the value back to rem
```

Comment:
This is my inellegent way to convert a Formation `rem` value to a USWDS `rem` value.

Reply:
Ray's version is much much better: https://github.com/department-of-veterans-affairs/vets-website/pull/26440